src/@chakra-ui/gatsby-plugin/components/Badge.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentStyleConfig, SystemStyleObject } from "@chakra-ui/react"
+
+const variantSecondary: SystemStyleObject = {
+  borderColor: "primary100",
+  color: "text",
+}
+
+export const Badge: ComponentStyleConfig = {
+  baseStyle: {
+    borderRadius: "base",
+    border: "1px solid",
+    borderColor: "transparent",
+    color: "black300",
+    fontWeight: "initial",
+    py: 1,
+    px: 2,
+    textTransform: "uppercase",
+  },
+  variants: {
+    secondary: variantSecondary,
+  },
+  sizes: {
+    sm: {
+      py: 0,
+    },
+    lg: {
+      px: 3,
+    },
+  },
+  defaultProps: {
+    // Remove the default from Chakra
+    variant: "",
```

Comment:
I'm wondering if we should have a default variant. IMO we should, since that is what we implicitly have right now on the designs. I think we should give it a name and put some defaults like the default background color.

I think that that variant should be the `solid` variant, having something like this:

```tsx
baseStyle: {
  borderRadius: "base",
  border: "1px solid",
  borderColor: "transparent",
  fontWeight: "initial",
  py: 1,
  px: 2,
  textTransform: "uppercase",
},
variants: {
  solid: {
    color: "black300",
    backgroundColor: "gray.300",
  },
  ...
},
defaultProps: {
  variant: "solid"
}
```

LMK what do you think.

Think should also fix the transparent bg we have on the l2 page:
![image](https://user-images.githubusercontent.com/468158/212396933-8d44f235-def9-4ad4-91da-0ecebaf8781c.png)